lms/envs/test.py

```diff
@@ -616,3 +616,12 @@
 MANDRILL_API_KEY = AUTH_TOKENS.get("MANDRILL_API_KEY")
 MAILCHIMP_API_KEY = AUTH_TOKENS.get("MAILCHIMP_API_KEY")
 MAILCHIMP_LEARNERS_LIST_ID = ""
+
+with open(CONFIG_ROOT / CONFIG_PREFIX + "env.json") as env_file:
+    ENV_TOKENS = json.load(env_file)
+
+if ENV_TOKENS.get('COMPREHENSIVE_THEME_DIR', None):
```

Comment:
We can modify this check from 
`if ENV_TOKENS.get('COMPREHENSIVE_THEME_DIR', None):` to `if ENV_TOKENS.get('COMPREHENSIVE_THEME_DIR'):` since dict.get returns None by default but the sanity check is need so that we set COMPREHENSIVE_THEME_DIR only in the case where it is given in the environment